src/dlgprefreplaygaindlg.ui

```diff
@@ -21,22 +21,22 @@
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
-       <widget class="QCheckBox" name="EnableAnalyser">
+       <widget class="QCheckBox" name="EnableGain">
         <property name="toolTip">
-         <string>Calculate ReplayGain normalization for tracks which are missing ReplayGain metadata.</string>
+         <string>Apply loudness normalization to loaded tracks.</string>
         </property>
         <property name="text">
-         <string>Enable ReplayGain Analysis</string>
+         <string>Apply ReplayGain</string>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="QCheckBox" name="EnableGain">
+       <widget class="QCheckBox" name="EnableAnalyser">
         <property name="toolTip">
-         <string>Apply loudness normalization to loaded tracks.</string>
+         <string>Calculate ReplayGain normalization for tracks which are missing ReplayGain metadata.</string>
         </property>
         <property name="text">
-         <string>Apply ReplayGain</string>
+         <string>Enable ReplayGain 1.0 Analysis</string>
```

Comment:
I don't think the version number is important, especially since 2.0 is supposed to be backward-compatible.